zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -309,15 +315,27 @@ pub fn init_blocks_table<P: consensus::Parameters>(
 
         wdb.conn.0.execute(
             "INSERT INTO blocks (height, hash, time, sapling_tree)
-            VALUES (?, ?, ?, ?)",
-            [
-                u32::from(height).to_sql()?,
-                hash.0.to_sql()?,
-                time.to_sql()?,
-                sapling_tree.to_sql()?,
+             VALUES (:height, :hash, :time, :sapling_tree)",
+            named_params![
+                ":height": u32::from(height),
+                ":hash": hash.0,
+                ":time": time,
+                ":sapling_tree": sapling_tree,
             ],
         )?;
 
+        if let Some(sapling_activation) = wdb.params.activation_height(NetworkUpgrade::Sapling) {
+            wdb.conn.0.execute(
+                "INSERT INTO scan_queue (block_range_start, block_range_end, priority)
+                 VALUES (:block_range_start, :block_range_end, :priority)",
+                named_params![
+                   ":block_range_start": u32::from(std::cmp::min(sapling_activation, height)),
```

Comment:
```suggestion
                   ":block_range_start": u32::from(start_height),
```
(if the above suggestion is taken).